examples/raycast.js

```diff
@@ -1,11 +1,14 @@
-const mineflayer = require('mineflayer')
+import { createBot } from 'mineflayer'
 
 if (process.argv.length < 4 || process.argv.length > 6) {
   console.log('Usage : node raycast.js <host> <port> [<name>] [<password>]')
   process.exit(1)
 }
 
-const bot = mineflayer.createBot({
+console.log('Commands :\n' +
+  '  have block in message')
```

Comment:
Why console log? It's not intended to be used as a bot. It's intended to act as an example of a bot. Anyone who use that file will look at the code before running it.